tests/providers/amazon/aws/log/test_cloudwatch_task_handler.py

```diff
@@ -154,49 +154,22 @@ def test_read(self):
             [{"end_of_log": True}],
         )
 
-    def test_read_wrong_log_stream(self):
```

Comment:
I deleted these 2 test cases for the following reasons.
* They overlap with the one I added (testing for generic failure to fetch remote logs)
* AWS error messages are now propagated to the task log. These strings could change in the future and break our unit tests.